app/controllers/api/v1/search_results_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SearchResultsController < ApplicationController
+      before_action :authorize!
+
+      def index
+        word = search_params[:adwords_url_contains]
```

Comment:
Optional idea: `word` is concise but a bit generic, we can add one more word to better describing it `word_params` or `word_query`... or you can propose your own.

```suggestion
        word_params = search_params[:adwords_url_contains]
```